Apps/W1/Shopify/app/src/Companies/Codeunits/ShpfyCompanyAPI.Codeunit.al

```diff
@@ -68,6 +68,10 @@ codeunit 30286 "Shpfy Company API"
         GraphQuery := CreateGraphQueryUpdateLocation(CompanyLocation);
         if GraphQuery <> '' then
             JResponse := CommunicationMgt.ExecuteGraphQL(GraphQuery);
+
+        CreateGraphQueryUpdateCompanyLocationTaxId(CompanyLocation);
```

Comment:
Confusing names. It says to create a query, but it actually send a query already.
Just UpdateCompanyLocationTaxId maybe?